### ISSUE TITLE
admin user better use different account and allow_ips

### ISSUE THREAD

**Issue by martianzhang** [flike/kingshard#161]
Now admin user and application user use the same user config.
It's hard for DBA to limit application's behaviour.

**Comment by flike** [comment]
yeah, I have have realised this problem. May I will different port for admin user. Thank you for you feedback. :)
